frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditTab/utils/utils.ts

```diff
@@ -0,0 +1,62 @@
+import type {
+  FormComponent,
+  SelectionComponentType,
+} from '../../../../../../../types/FormComponent';
+import type { FormItem } from '../../../../../../../types/FormItem';
+import type { FormContainer } from '../../../../../../../types/FormContainer';
+import type { Option } from 'app-shared/types/Option';
+
+export function handleOptionsChange(
+  updatedComponent: FormItem<SelectionComponentType>,
+  handleComponentChange: (item: FormContainer | FormComponent) => void,
+): void {
+  handleComponentChange(updatedComponent);
+}
+
+export function resetComponentOptions(
+  component: FormItem<SelectionComponentType>,
+): FormItem<SelectionComponentType> {
+  const newComponent = { ...component };
```

Comment:
Nitpicking, but when doing this we can use the `ObjectUtils.deepCopy`. That util function will also create new objects if there are changes in nested properties in the object.